src/CommunityToolkit/Xamarin.CommunityToolkit/Views/GravatarImageSource/Shared/GravatarImageSourceHandler.android.ios.macos.tizen.uwp.wpf.cs

```diff
@@ -87,12 +88,20 @@ static string CacheFileName(GravatarImageSource gis, float scale)
 		static async Task<byte[]> GetGravatarAsync(string email, int size, float scale, DefaultGravatar defaultGravatar)
 		{
 			var requestUri = GetGravatarUri(email, size, scale, defaultGravatar);
-			using var response = await lazyHttp.Value.GetAsync(requestUri);
+			try
+			{
+				using var response = await lazyHttp.Value.GetAsync(requestUri);
 
-			if (!response.IsSuccessStatusCode)
-				return Array.Empty<byte>();
+				if (!response.IsSuccessStatusCode)
+					return Array.Empty<byte>();
 
-			return await response.Content.ReadAsByteArrayAsync();
+				return await response.Content.ReadAsByteArrayAsync();
```

Comment:
Can this return `null`?

Reply:
looks like no.
![image](https://user-images.githubusercontent.com/33021114/155027261-bb90a944-c587-4f5b-a160-6e374b2d9994.png)